content/pages/kits.md

```diff
@@ -0,0 +1,72 @@
+
+​The DJ Table is a record player based system for demonstrating geophysical fluid dynamics. The system’s tanks come with transparent lids that make them ideal for visualization via phone or camera, and its minimal footprint makes for easy storage and transportation.​ ​
+
+### [Learn more about the DJ Table here](/pages/dj.html)
+
+---
+
+## HT3 Table - Starting at $5000
+
+{% img {static}../images/kits/ht3/ht3_table.PNG 400 HT3 Table complete setup %}
+
+The HT3 Table is a specially designed system including a large rotating tank, lightweight frame, and multi-functional cart that is meant to optimize geophysical fluid dynamics demonstrations while remaining accessible and portable.  This high-end device is ideal for demonstrating simple to advanced geophysical fluid processes, and can be easily stored and moved between locations.
+
+### [Learn more about the HT3 Table here](/pages/ht3.html)
+
+---
+
+## Helpful Tips
+
+Here are helpful tips when using any of the DIYnamics Kits. We greatly value feedback and contributions from the DIYnamics community to help refine our devices and experiments. If you have any suggestions, please feel free to share with us via email at diynamicsteam@gmail.com or Twitter at @DIYnamicsTeam.
+
+---
+
+### Recommended Parts
+
+In addition to the required components outlined in the order sheet, we also recommend the following items:
+
+- [GFCI Plug](https://www.amazon.com/dp/B000XVG72G/ref=cm_sw_em_r_mt_dp_aqZtFbF8WHY6S)
+
+**Disclaimer: Be careful with experiments involving wall power and water in close proximity.** The GFCI halts power in the event of a fault and is a vital safety measure in situations involving water and electricity.
+
+- [Bullseye Bubble Level](https://www.mcmaster.com/2147A11/)
+
+When working with the DIYnamics Kits or any other rotating tank platform, the tank must be level and center for optimal performance. Off-level or off-center tanks create unwanted fluid motions that detract from experiments. 
+
+- [Siphon Drum Pump](https://www.mcmaster.com/9894K44/)
+
+Using the DIYnamics Kits involves filling and draining water from tanks. Be prepared to handle water logistics when performing experiments.
+
+---
+
+### Capturing Rotating Frame Footage
+
+{% img {static}../images/kits/rotating_frame.png 400 What is the rotating frame? %}
+
+Audience members are encouraged to take a hands-on approach to observe and analyze fluid motions then tie them to the real world. However, viewers must contend with observing and analyzing fluid motions whilst abstracting away motion from rotation itself. Each DIYnamics Kit features options for capturing footage in the rotating frame or from a perspective inside the system looking down on the tank as it rotates. This feature is beneficial because it makes accessible fluid motions independent of rotation. Zoom can be used to livestream rotating frame footage for real-time analysis of fluid motions.
```

Comment:
When the tank on any of the DIYnamics kits are rapidly rotating, it can be hard to see the more subtle behaviors going on inside the tank.  To get around this, it's really helpful to have a camera looking at the tank from above and spinning with the tank.  Each DIYnamics Kit features options for doing this --- what we call capturing footage in the rotating frame.  Zoom can be used to livestream rotating frame footage for real-time analysis of fluid motions.